cloud-formation/explain-maker.cf.json

```diff
@@ -86,10 +86,7 @@
                       "#!/bin/bash -ev",
                       "/opt/features/ssh-keys/initialise-keys-and-cron-job.sh -l -b github-team-keys -t Digital-CMS || true \n",
                       "mkdir /etc/gu",
-                      "cat >/etc/gu/explainer.stage.conf << EOF",
-                      { "Fn::Join": [ "", [ "stage=", { "Ref": "Stage"} ] ] },
-                      "EOF",
-
+                      "aws s3 cp s3://explainer-config/application.secrets.conf /etc/gu/explainer.secrets.conf",
```

Comment:
I've found this to be really good for config, don't know if you've seen it? https://github.com/guardian/configuration-magic.